apps/src/templates/sectionsRefresh/CurriculumQuickAssignTopRow.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import Button from '@cdo/apps/templates/Button';
+import moduleStyles from './sections-refresh.module.scss';
+
+export const MARKETING_AUDIENCE = {
+  ELEMENTARY: 'elementary',
+  MIDDLE: 'middle',
+  HIGH: 'high',
+  HOC: 'hoc',
+  PL: 'pl'
+};
+
+export default function CurriculumQuickAssignTopRow({
+  isPl,
```

Comment:
I'm not sure what to call this. It's not necessarily whether or not we are *looking at PL offerings right now, it's more about whether or not PL should be available at all. Thoughts?

Reply:
Updating this now to showPlOfferings- is that better?

Reply:
makes sense to me!